src/genie/libs/parser/iosxe/show_hw_module_status.py

```diff
@@ -0,0 +1,116 @@
+''' show_hw_module_status.py
+IOSXE parsers for the following show commands:
+    * show hw_module subslot {subslot} transceiver {transceiver} status
+'''
+
+import re
+
+from genie.metaparser import MetaParser
+# from genie.metaparser.util.schemaengine import Any, Optional
+
+# parser utils
+# from genie.libs.parser.utils.common import Common
+
+
+# ==========================
+# Schema for:
+#  * 'show_hw_module_status'
+# ==========================
+class Show_Hw_Module_StatusSchema(MetaParser):
+    """Schema for show_hw_module_status."""
+
+    schema = {
```

Comment:
I think we can simplify more like below. and we usually have numbers as integer or float.
```
schema = {
    "slot_id": int,
    "subslot": int,
    "port_id": int,
    "module_temperature": float,
    "supply_voltage_mVolts": float,
    "bias_current_uAmps": int,
    "tx_power_dBm": float,
    "optical_power_dBm": float
}

Reply:
This has been updated to support float values.